<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingsWin</class>
 <widget class="QDialog" name="settingsWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>settingsWin</string>
  </property>
  <widget class="QTabWidget" name="tabWidgetSettings">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>571</width>
     <height>351</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="interfaceTab">
    <attribute name="title">
     <string>Interface</string>
    </attribute>
    <widget class="QLabel" name="tabsPosLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Tabs position:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="tabsPosCombo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Top (default)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bottom</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Left</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Right</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="tabsShapeLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Tabs Shape:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="tabsShapeCombo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Rounded (default)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Triangular</string>
      </property>
     </item>
    </widget>
    <widget class="QRadioButton" name="yesRadioBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>96</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Yes</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="noRadioBtn">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>96</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>No</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>191</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Show vertical headers on tables:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="businessTab">
    <attribute name="title">
     <string>Business</string>
    </attribute>
    <widget class="QLineEdit" name="nameLineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>241</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Acme Co</string>
     </property>
    </widget>
    <widget class="QPushButton" name="browseBtn">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>250</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>Browse...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="phoneLineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>241</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>(123) 456-7890</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="emailLineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>241</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>contact@acme.com</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="addressLineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>241</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>99 Xyz Street</string>
     </property>
    </widget>
    <widget class="QLabel" name="nameLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="phoneLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Phone:</string>
     </property>
    </widget>
    <widget class="QLabel" name="emailLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>E-mail:</string>
     </property>
    </widget>
    <widget class="QLabel" name="addressLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Address:</string>
     </property>
    </widget>
    <widget class="QLabel" name="logoLabel">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Logo:</string>
     </property>
    </widget>
    <widget class="QLabel" name="businessLogo">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>40</y>
       <width>200</width>
       <height>200</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <property name="indent">
      <number>-1</number>
     </property>
     <property name="openExternalLinks">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="saveBtn">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>370</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="icon">
    <iconset theme="document-save"/>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>370</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="icon">
    <iconset theme="edit-delete"/>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>phoneLineEdit</tabstop>
  <tabstop>emailLineEdit</tabstop>
  <tabstop>addressLineEdit</tabstop>
  <tabstop>saveBtn</tabstop>
  <tabstop>tabsShapeCombo</tabstop>
  <tabstop>yesRadioBtn</tabstop>
  <tabstop>tabWidgetSettings</tabstop>
  <tabstop>tabsPosCombo</tabstop>
  <tabstop>noRadioBtn</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>nameLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>phoneLineEdit</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>phoneLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>emailLineEdit</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>emailLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>addressLineEdit</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addressLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>saveBtn</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>557</x>
     <y>380</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
